### PR TITLE
Fix help-debug -checkpoints

### PR DESCRIPTION
**Pull request by ariard**

### CODE REVIEW

File: src/validation.h [hunk 1]
```diff
@@ -105,17 +105,22 @@ static const int64_t BLOCK_DOWNLOAD_TIMEOUT_BASE = 1000000;
 /** Additional block download timeout per parallel downloading peer (i.e. 5 min) */
 static const int64_t BLOCK_DOWNLOAD_TIMEOUT_PER_PEER = 500000;
 
+/** Default for -maxtipage, maximum tip age in seconds to consider node in initial block download. */
 static const int64_t DEFAULT_MAX_TIP_AGE = 24 * 60 * 60;
 /** Maximum age of our tip in seconds for us to be considered current for fee estimation */
 static const int64_t MAX_FEE_ESTIMATION_TIP_AGE = 3 * 60 * 60;
 
+/** Default for -checkpoints, enable skipping verification of known chain history until block 295000. */
```
Comment (jnewbery) [inline src/validation.h]: That's not what checkpoints do :slightly_smiling_face:

Comment (MarcoFalke) [reply src/validation.h]: To be fair the `-help-debug` was never updated either

Comment (MarcoFalke) [reply src/validation.h]: See:

* #16551

Comment (ariard) [reply src/validation.h]: Oh you're right, I've double-down on the error already there in the help-debug. Fixed both

**Comment by laanwj** [comment]
Thanks for adding documentation, though I'm not sure it's a good idea to duplicate command line option documentation when documenting defaults. This will tend to go out of sync. ~0 on this.

**Comment by MarcoFalke** [comment]
Might be fine to just say "default for -foobar"

**Comment by ariard** [comment]
@laanwj yes I was a bit dubious at first because as you they can fall out of sync easily but then I noticed than some of them were already commented like `DEFAULT_MIN_RELAY_TX_FEE` and `DEFAULT_ANCESTOR_LIMIT`. Was willingly to be consistent there, and that's still less code to grep when you're trying to understand what these values are doing.

**Comment by jnewbery** [comment]
I'm with @laanwj on this one. Duplicating comments seems unnecessary, and adds the potential for them to go out of sync.

**Comment by MarcoFalke** [comment]
`git grep -C 1 '\-persistmempool\>'  src` can be used to display the default value and the help text, so no need to duplicate it

**Comment by ariard** [comment]
Okay no worries, curtailed commit to just fixing -checkpoints help-debug if people don't think duplicating comment is necessary.

**Comment by jnewbery** [comment]
I think the `default for -option` code comments are useful, just not repeating the help text. I'd support you adding those.

ACK a8b82867d544d221de00f6a0e26dbcc389326629 for improving the `-prune` help text.

**Comment by MarcoFalke** [comment]
ACK a8b82867d544d221de00f6a0e26dbcc389326629

**Pull request merged**